docs/source/automol/matrix.md

```diff
@@ -0,0 +1,5 @@
+# automol.matrix
```

Comment:
See comment for `heuristic.md` I want this `automodule` block moved into `util.md` so that all of the util functions are in one place.